k8s/prod-deployment.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
```

Comment:
Please use the same deploy.yml and service.yml files as base for every environment.

Reply:
Hi, I already done, please you can review.